src/test/java/com/hocs/test/pages/decs/SummaryTab.java

```diff
@@ -385,6 +385,13 @@ public void assertAllocatedTeam(String team) {
         allocatedTeam.shouldContainText(team);
     }
 
+    public void assertPreviousCaseRef(String prevCaseRef) {
```

Comment:
Could we name this 'assertPreviousCaseReferenceIsVisible', just so its more readable for future reference.